faucet/valve.py

```diff
@@ -1036,10 +1036,15 @@ def lldp_handler(self, now, pkt_meta, other_valves):
                 self.logger.info('LLDP on %s, %s from %s (remote %s, port %u) state %s' % (chassis_id, port, pkt_meta.eth_src, valve_util.dpid_log(remote_dp_id), remote_port_id, remote_port_state))
             port.dyn_lldp_beacon_recv_state = remote_port_state
 
+        exp_remote_src = self.dp.ports[port.number].lldp_peer_mac
+        self.logger.info('Anurag debug config: %s peer: %s' % (exp_remote_src, pkt_meta.eth_src))
+        if exp_remote_src and exp_remote_src != pkt_meta.eth_src:
```

Comment:
only consideration here is that this will be a very spammy message -- every LLDP packet. we might consider moving it up into the state-protected clause, so it's only triggered if the state change (few lines above).  Honestly not sure which is better. spammy logs seem like a good idea (make warning obvious), but sometimes really get in the way of another more-important "real" issue.  I personally would move it to the protected clause, and then rely on varz monitoring added later to make the error more obvious.